dts/arm/st/u5/stm32u5.dtsi

```diff
@@ -149,6 +150,7 @@
 			compatible = "st,stm32-exti";
 			interrupt-controller;
 			#interrupt-cells = <1>;
+			#address-cells = <1>;
```

Comment:
Thanks for fixing this, but please do it in a dedicated commit (mentioning the warning you fix).
Pushing a bit, even a dedicated PR would be nice so we can discuss more around this, as I'm not seeing this warning on my side, and maybe fix on all series.

Reply:
Sure, will create separate commits.

I can add the configuration for the B-U585I-IOT02A. I just saw, it has additional SRAM available. However, I will not be able to test it myself.

Reply:
> On the B-U585I-IOT02A, the SRAM is connected via SPI and not via FMC. The nucleo_u575zi_q also does not have any FMC connected hardware

Ok, then let it aside.

Reply:
Alright, I removed the last commit. I think this request can be resolved now?

Reply:
Seems it reappeared somehow